src/auto-evo/simulation/PopulationSimulation.cs

```diff
@@ -206,10 +234,29 @@ private static List<Species> CopyInitialPopulationsToResults(SimulationConfigura
                 var newPopulation = (long)(energyBySpecies[currentSpecies]
                     / energyBalanceInfo.FinalBalanceStationary);
 
-                // Severely penalize a species that can't osmoregulate
-                if (energyBalanceInfo.FinalBalance < 0)
+                if (trackEnergy)
+                {
+                    populations.AddTrackedEnergyConsumptionForSpecies(currentSpecies, patch, newPopulation,
+                        energyBySpecies[currentSpecies], energyBalanceInfo.FinalBalanceStationary);
+                }
+
+                // TODO: this is a hack for now to make the player experience better, try to get the same rules working
+                // for the player and AI species in the future.
+                if (currentSpecies.PlayerSpecies)
                 {
-                    newPopulation /= 10;
+                    // Severely penalize a species that can't osmoregulate
+                    if (energyBalanceInfo.FinalBalanceStationary < 0)
+                    {
+                        newPopulation /= 10;
+                    }
+                }
+                else
+                {
+                    // Severely penalize a species that can't move indefinitely
+                    if (energyBalanceInfo.FinalBalance < 0)
+                    {
+                        newPopulation /= 10;
+                    }
```

Comment:
Shouldn't we also add in some sessility factor? Otherwise that could be very detrimental to AI plant species.

Reply:
Maybe but that'll have to wait for another PR.